test/runtests.jl

```diff
@@ -7,7 +7,7 @@ using Test
 
 @testset "BlockDiagonals" begin
     # The doctests fail on x86, so only run them on 64-bit hardware
-    Sys.WORD_SIZE == 64 && doctest(BlockDiagonals)
+    # Sys.WORD_SIZE == 64 && doctest(BlockDiagonals)
```

Comment:
Why is this commented out rather than properly removed?

Reply:
Opps that was just me diabling docttests to make it run faster
```suggestion
    Sys.WORD_SIZE == 64 && doctest(BlockDiagonals)
```